src/containers/editor/graph/Toolbar.tsx

```diff
@@ -59,12 +61,10 @@ const Toolbar = memo(({ id }: ToolbarProps) => {
     setFoldSiblings(!foldSiblings);
   };
 
-  // TODO: hide fold button when there is no edges
-  // TODO: fix w-fit doesn't work
   return (
     <NodeToolbar
       isVisible={true}
-      className="flex items-center justify-center w-[96px] h-fit bg-input"
+      className="flex items-center justify-center w-fit h-fit px-2 bg-input"
```

Comment:
If we add padding to the button group, it might mislead the user because the real clickable area is smaller than what the user sees.

Reply:
Looks like padding for button seems an issue. I replaced it with gap in the container. Can you review it?

Reply:
This is reverted. Thank you